### PR TITLE
Mirego-base-setup 2 of X: Bootstrapper + app initialization

### PR DESCRIPTION
**Pull request by OlivierPineau**
## 📖 Description
Add bootstrapper to the shared code + setup iOS and Android platform initialization 
The bootstrapper is used at the initialization of the app to setup the Root viewmodel and the dependency injection framework, Koin

## 🦀 Dispatch

- `#dispatch/kmp`

### CODE REVIEW

File: ios/iosApp/AppInitializer.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import Trikot
+
+enum AppInitializer {
+    static func initializeComponents(environment: AppEnvironment) {
+        initializeCommon()
+        initializeKingfisher()
+    }
+
+    private static func initializeCommon() {
+        TrikotKword.shared.setCurrentLanguage(Foundation.Locale.isPreferredLanguagesFrench ? "fr" : "en")
```
Comment (marcantoinefortier) [inline ios/iosApp/AppInitializer.swift:13]: This should be done in the common code using expect-actual implementations providing the current language code.

File: ios/iosApp/Domain/AppInformationImpl.swift [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import Foundation
+import Shared
+
+class AppInformationImpl: AppInformation {
```
Comment (marcantoinefortier) [inline ios/iosApp/Domain/AppInformationImpl.swift:4]: This should also be done using expect-actual instead of platform specific implementations

Comment (sdetilly) [reply ios/iosApp/Domain/AppInformationImpl.swift:4]: The problem here is that on android, we need the Context object and access to the buildConfig of the app. If we do this in the `androidMain` in the `shared` module, we don't have access to any of those.

Comment (marcantoinefortier) [reply ios/iosApp/Domain/AppInformationImpl.swift:4]: I got a solution for this that I'll put on main branch 🙂

**Pull request merged**